app/src/main/java/de/grobox/transportr/networks/TransportNetworks.kt

```diff
@@ -458,13 +445,6 @@ private val networks = arrayOf(
             ),
             Country(
                 R.string.np_region_poland, "🇵🇱", networks = listOf(
-                    TransportNetwork(
-                        id = NetworkId.PL,
-                        name = R.string.np_name_pl,
-                        description = R.string.np_desc_pl,
-                        logo = R.drawable.network_pl_logo,
```

Comment:
If you remove networks, please also remove the associated resources such as strings and drawables. Then the string update for Poland can probably removed from changelog.

Reply:
Good catch. I removed the related unused resources.